pkg/scaffold/scaffold.go

```diff
@@ -58,6 +58,9 @@ type Scaffold struct {
 
 	// Plugins is the list of plugins we should allow to transform our generated scaffolding
 	Plugins []Plugin
+
+	// Indicates that the Scaffold should respect the multigrup structure
+	Multigrup bool
```

Comment:
Typo that shows that you are not using this field as it would have given an error.